src/index.js

```diff
@@ -3,12 +3,14 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  // <React.StrictMode>
-  <App />,
-  // </React.StrictMode>,
+  <BrowserRouter>
+   <App />
+  </BrowserRouter>
+ 
 );
 
 // If you want to start measuring performance in your app, pass a function
```

Comment:
- [OPTIONAL] I suggest that you remove comments from your code this would ensure you have a cleaner codebase